esw/readme.md

```diff
@@ -0,0 +1,12 @@
+# ESW Ontology: Exploratory Search Workflow
+
+This [W3ID](https://w3id.org) provides a persistent URI namespace for Exploratory Search Workflow (ESW) resources.
+
+## Contact
+This space is administered by:  
+
+**Gianmarco Prando**  
+*Computer Engineer *
+Padua, Italy  
```

Comment:
To make the markup and line breaks behave as you intended... you'll want to make another PR, with these lines --
```suggestion
**Gianmarco Prando**  
*Computer Engineer*  
Padua, Italy  
```